html5validator/cli.py

```diff
@@ -90,8 +93,8 @@ def main():
     args, extra_args = parser.parse_known_args()
 
     if args.config is not None:
-        args = parse_yaml(args.config, args)
-        LOGGER.debug(args)
+        args, extra_args = parse_yaml(args.config, args)
+        LOGGER.debug(args, extra_args)
```

Comment:
Don't you need a format string as the first argument?
At this point, logging is also not configured yet. Maybe remove.

Reply:
I removed the logging here. Don't know what you refer to with format string and the first argument in the path to the config file.